plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -124,7 +128,22 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
       left: 20,
       right: 20,
     },
-    xAxis: { type: 'time' },
+    xAxis: { 
+      type: 'time', 
+      axisLabel: { 
+        showMinLabel: !!xAxisShowMinLabel,
+        showMaxLabel: !!xAxisShowMaxLabel,
+        ...(xAxisFormat && {
+          formatter: value => {
+            if (moment(value).isValid()) {
+              return moment(value).format(xAxisFormat);
+            }
+
+            return moment(value).format('YYYY-MM-DD')
+          },
+        })
```

Comment:
We should not hardcode the format, but rather add a similar control and functionality in Pivot Table. See https://github.com/apache-superset/superset-ui/blob/master/plugins/legacy-plugin-chart-pivot-table/src/PivotTable.js#L68-L74